TOWebViewController/TOWebViewController.m

```diff
@@ -729,6 +729,14 @@ - (void)setButtonTintColor:(UIColor *)buttonTintColor
     }
 }
 
+- (void)setDoneButtonTintColor:(UIColor *)doneButtonTintColor
+{
+    if (doneButtonTintColor == _doneButtonTintColor)
+        return;
+    
+    _doneButtonTintColor = doneButtonTintColor;
+}
+
```

Comment:
This method isn't necessary since it's not doing anything extra after updating the property value. You can probably just delete it as is, and nothing will change. :)